example-scenegraph-sdk/source/components/helloworld.brs

```diff
@@ -28,62 +28,62 @@ sub init()
     consentState = consentStateAPI.build()
     
     ' GDPR
-    gdprConsentStateApi = mpConstants.GDPRConsentState
-    gdprConsentState = gdprConsentStateApi.build(False, time.asSeconds())
+    'gdprConsentStateApi = mpConstants.GDPRConsentState
```

Comment:
perhaps you can add a comment that says something like...

```
This portion is commented out because a workspace has to be configured for specific GDPR purposes (in this case, 'functional').  If the purpose is not configured, events will not be accepted and not show up in the Live Stream. To test your own implementation, make sure you have a GDPR consent purpose activated on your workspace and update the purpose below.  Similarly, CCPA (which does not require a purpose) must be configured on your workspace for events with a CCPA attached to work.
```